runtime/js/40_fs_events.js

```diff
@@ -21,13 +20,6 @@ class FsWatcher {
   #promise;
 
   constructor(paths, options) {
-    if (internals.future) {
-      ObjectDefineProperty(this, "rid", {
-        __proto__: null,
-        enumerable: false,
-        value: undefined,
-      });
```

Comment:
This needs to stay

Reply:
This is news to me. Why?

Reply:
Nvm, i read it wrong. Yeah this one goes.